src/test/java/htsjdk/samtools/util/IoUtilTest.java

```diff
@@ -179,6 +182,13 @@ public void testAddExtension() throws IOException {
         Assert.assertEquals(IOUtil.addExtension(p, ".ext"), IOUtil.getPath("/folder/file.ext"));
         p = IOUtil.getPath("folder/file");
         Assert.assertEquals(IOUtil.addExtension(p, ".ext"), IOUtil.getPath("folder/file.ext"));
+        FileSystem jimfs = Jimfs.newFileSystem(Configuration.unix());

```

Comment:
The `FileSystem` should be closed. I guess that try-with-resources is the better option for that...